mm2src/coins/utxo/utxo_block_header_storage/sql_block_header_storage.rs

```diff
@@ -87,9 +87,16 @@ fn get_block_height_by_hash(for_coin: &str) -> Result<String, BlockHeaderStorage
     Ok(sql)
 }
 
-fn remove_headers_up_to_height_sql(for_coin: &str, to_height: u64) -> Result<String, BlockHeaderStorageError> {
+fn remove_headers_from_to_height_sql(
+    for_coin: &str,
+    from_height: &u64,
+    to_height: &u64,
+) -> Result<String, BlockHeaderStorageError> {
     let table_name = get_table_name_and_validate(for_coin)?;
-    let sql = format!("DELETE FROM {table_name} WHERE block_height <= {to_height};");
+    let sql = format!(
+        "DELETE FROM {table_name} WHERE block_height BETWEEN {from_height} AND 
+    {to_height};"
+    );
```

Comment:
why not this?
```suggestion
    let sql = format!("DELETE FROM {table_name} WHERE block_height BETWEEN {from_height} AND {to_height};");
```

Reply:
Please prefer using prepared statements here.

Reply:
done

Reply:
can you please check if the latest commit solves the problem?

Reply:
it solves, thank you